### DISCUSSION TITLE
Organization address in the invoice

### DISCUSSION THREAD

**Discussion by Fulcrum99** [community/community#65145] [New to GitHub]
hello Support,

We want the organization address in the invoice. How to get it? ..

**Comment by MaroIsLife** [comment]
Normally it's added in the payement information 

https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-information-to-your-receipts

**Comment by tinohager** [comment]
Are there plans to bring the bill a little more into a format?
https://www.google.com/search?q=invoice+template+america&tbm=isch
